cannon/mipsevm/exec/mips_instructions.go

```diff
@@ -1,96 +1,114 @@
 package exec
 
 import (
+	"fmt"
+
 	"github.com/ethereum-optimism/optimism/cannon/mipsevm"
+	"github.com/ethereum-optimism/optimism/cannon/mipsevm/arch"
 	"github.com/ethereum-optimism/optimism/cannon/mipsevm/memory"
+
+	// TODO: MIPS64 port. Replace with a custom library
```

Comment:
Would be good to create an issue for this and reference it in the TODO.

Reply:
Indeed. I've created an issue to track these and referenced the ticket in the TODOs.